libbindgen/src/ir/function.rs

```diff
@@ -147,6 +147,7 @@ impl FunctionSig {
         };
         let mut args: Vec<_> = match cursor.kind() {
             CXCursor_FunctionDecl |
+            CXCursor_Constructor |
             CXCursor_CXXMethod => {
```

Comment:
Is there a reason this doesn't also match `CXCursor_Destructor` like the `ClangSubItemParser` impl further down does?

Is it handled elsewhere?